tests/test/baseledger.test.js

```diff
@@ -613,6 +613,150 @@ describe('baseledger transaction', async function() {
   });
 });
 
+describe('bad input', async function() {
+  this.timeout(70000);
+  before(() => {
+    startTestNet();
+  });
+
+  after(() => {
+    cleanTestNet();
+  });
+
+  it('should pass bad deposit address and not increase balance', async function() {
```

Comment:
both tests start with bad input, and the with good, checking if event nonces recovered properly